test/e2e-tests.sh

```diff
@@ -46,7 +46,7 @@ TEST_YAML_IGNORES=${TEST_YAML_IGNORES:-""}
 
 # Allow ignoring some yaml tests, space separated, should be the basename of the
 # test for example "s2i"
-TEST_TASKRUN_IGNORES=${TEST_TASKRUN_IGNORES:-"docker-build"}
```

Comment:
we should not skip kaniko

Reply:
for now to make the CI green I am skipping the kaniko but will fix that in #485